mqtt.kafka.asyncapi.proxy/zilla.yaml

```diff
@@ -2,324 +2,13 @@
-                        'streetlights:on': Ability to switch lights on
-                        'streetlights:off': Ability to switch lights off
-                        'streetlights:dim': Ability to dim the lights
-                openIdConnectWellKnown:
-                  type: openIdConnect
-                  openIdConnectUrl: 'https://authserver.example/.well-known'
-              parameters:
-                streetlightId:
-                  description: The ID of the streetlight.
-              messageTraits:
-                commonHeaders:
-                  headers:
-                    type: object
-                    properties:
-                      my-app-header:
-                        type: integer
-                        minimum: 0
-                        maximum: 100
-              operationTraits:
-                mqtt:
-                  bindings:
-                    mqtt:
-                      qos: 1
+          path: mqtt/mqtt-asyncapi.yaml
         kafka:
-          schema: |
-            asyncapi: 3.0.0
-            info:
-              title: Zilla Kafka Proxy
-              version: 1.0.0
-              license:
-                name: Aklivity Community License
-            servers:
-              plain:
-                host: host.docker.internal:9092
-                protocol: kafka
-            
-            operations:
-              onStreetlightData:
-                action: receive
-                channel:
-                  $ref: '#/channels/streetlightData'
-              toStreetlightData:
-                action: send
-                channel:
-                  $ref: '#/channels/streetlightData'
-            
-            channels:
-              streetlightData:
-                description: This channel contains a message for streetlights.
-                address: streetlights
-                messages:
-                  streetlightData:
-                    $ref: '#/components/messages/streetlightData'
-              mqttSessions:
-                description: This channel contains MQTT sessions.
-                address: mqtt-sessions
-              mqttMessages:
-                description: This channel contains MQTT messages.
-                address: mqtt-messages
-              mqttRetained:
-                description: This channel contains MQTT retained messages.
-                address: mqtt-retained
-            
-            components:
-              messages:
-                streetlightData:
-                  payload:
-                    type: object
-                    properties:
-                      streetlightId:
-                        type: integer
-                        description: This property describes the id of the streetlight
-                      message:
-                        type: string
-                        description: This property describes message of the streetlight
+          path: kafka/kafka-asyncapi.yaml
 bindings:
   asyncapi_mqtt0:
```

Comment:
can you update any binding names ending in `0` to be more specific to what the binding is doing. This one can be the `north_mqtt_server`. and the kafka proxy can be `south_kafka_proxy`.